packages/admin-ui/src/tag/tag.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { render, axe } from '../test-utils'
+import { Tag } from './tag'
+
+const LABEL = 'Rio de Janeiro'
+
+describe('Tag tests', () => {
+  it('should have overridable styles', () => {
+    const { getByTestId } = render(
+      <Tag label="tag-1" data-testid="tag" csx={{ bg: 'coral' }} />
+    )
+
+    expect(getByTestId('tag')).toHaveStyleRule('background', 'coral')
+  })
+
+  it('should match snapshot', () => {
+    const { asFragment } = render(
+      <>
+        <Tag label={LABEL} variant="cyan" />
+        <Tag label={LABEL} variant="purple" />
+        <Tag label={LABEL} variant="green" />
+        <Tag label={LABEL} variant="red" />
+        <Tag label={LABEL} variant="purple" />
+        <Tag label={LABEL} variant="orange" />
+        <Tag label={LABEL} variant="teal" />
+        <Tag label={LABEL} variant="gray" />
+        <Tag label="tag-1" />
+      </>
+    )
+
+    expect(asFragment()).toMatchSnapshot()
+  })
```

Comment:
We'll discontinue snapshot tests, IMO it's ok to remove it

Reply:
Roger that 😉